Bootstrap-social.min.css updated to 4.3.3

dalanzg opened the pull request:
I have just updated bootstrap-social.min.css to 4.3.3. The file was copied from the original repository: https://github.com/lipis/bootstrap-social

I added both versions and modified the package.json file. This is my first contribution to cdnjs, so please, let me know if I performed the steps correctly.

Regards,

Dalanzg

PeterDaveHello commented:
Please squash these commits into only one commit and auto-update config as #3638 said, thanks.

dalanzg commented:
Hello, I do not know how to squash these three commits. I only did the following:

1. I created the folder cdnjs/ajax/libs/bootstrap-social/4.3.3
2. I added bootstrap-social.css and bootstrap-social.min.css from https://github.com/lipis/bootstrap-social
3. I modified package.json (version 4.3.3)

Regarding the comment "Added Pinterest (Fixes #35)", it is related to the modification of the CSS file. It was added the Social Icon from Pinterest (It is not related to CDNJS repository).

Could you help me to end the pull request? I only get a message:  Able to merge. These branches can be automatically merged.

Thanks in advance

PeterDaveHello commented:
It only meant there is no conflict and pass the CI build, didn't mean anything else, like policy and convention, if the comment `Added Pinterest (Fixes #35)` is not related to cdnjs repo, you should remove it or mention to the related repo by GitHub flavor, as you can see, if you click it, it'll link to #35 in cdnjs.
